### ISSUE TITLE
Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller

### ISSUE THREAD

**Issue by parismiguel** [PowerShell/PSReadLine#2977]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
Oops, something went wrong.
Please report this bug with ALL the details below, including both the 'Environment' and 'Exception' sections.
Please report on GitHub: https://github.com/PowerShell/PSReadLine/issues/new?template=Bug_Report.md
Thank you!

### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.19042
BufferWidth: 120
BufferHeight: 30

Last 0 Keys


```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot

N/A

### Environment data

```console
PS Version: 7.2.0
PS HostName: ConsoleHost (Windows Terminal)
PSReadLine Version: 2.2.0-beta2
PSReadLine EditMode: Windows
OS: 10.0.19041.1 (WinBuild.160101.0800)
BufferWidth: 120
BufferHeight: 30
```

### Steps to reproduce

Just try to open PowerShell

### Expected behavior

Get the prompt

### Actual behavior

System breaks

**Comment by sonnypate** [comment]
I'm seeing the same error after updating to Powershell 7.2.0:

### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.19043
BufferWidth: 150
BufferHeight: 45

Last 0 Keys
```

```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

**Comment by sonnypate** [comment]
Actually it looks like this issue is fixed by installing PSReadLine 2.2.0-beta4. I was originally using 2.2.0-beta2. Since I could not get past the error to update the module, I had to remove the PSReadLine folder from my Powershell profile, then I was able to install the new version using 'install-module PSReadLine -AllowPrerelease -Force'.

**Comment by iSazonov** [comment]
Yes, please update to Beta4 version.

**Comment by parismiguel** [comment]
Removing folder PSReadLine from here fixed it. Thanks @glitch32 

![image](https://user-images.githubusercontent.com/8812746/140954511-4f5f562b-a9be-49bc-9e60-69e20682c981.png)

**Comment by Harfeur** [comment]
> Removing folder PSReadLine from here fixed it. Thanks @glitch32
> 
> ![image](https://user-images.githubusercontent.com/8812746/140954511-4f5f562b-a9be-49bc-9e60-69e20682c981.png)

It's not fixed for me, I still have the error after deleting the folder

**Comment by Harfeur** [comment]
Issue #2974 fixed it using `pwsh.exe -noprofile -command "Install-Module PSReadLine -Force -SkipPublisherCheck -AllowPrerelease"`

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.

**Comment by hananiah-linde** [comment]
> 
> 
> This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
> 
>     1. stop all instances of `pwsh`.
> 
>     2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`
> 
> 
> If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
> 
>     1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
> 
>     2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.

Can confirm that this works :)
